templates/community/diversity.html

```diff
@@ -13,10 +14,10 @@ <h1>Diversity</h1>
   <div class="row">
     <div class="col-8">
       <p>The Ubuntu project welcomes and encourages participation by everyone. We are committed to being a community that everyone feels good about joining. Although we may not be able to satisfy everyone, we will always work to treat everyone well.</p>
-      <p>Standards for behaviour in the Ubuntu community are detailed in the <a href="/about/about-ubuntu/conduct">Code of Conduct</a>. We expect participants in our community to meet these standards in all their interactions and to help others to do so as well.</p>
+      <p>Standards for behaviour in the Ubuntu community are detailed in the <a href="/community/code-of-conduct">Code of Conduct</a>. We expect participants in our community to meet these standards in all their interactions and to help others to do so as well.</p>
```

Comment:
you are missing 'and Leadership Code of Conduct' with a link at the end of this sentence.